### DISCUSSION TITLE
How to initialization connect for server.

### DISCUSSION THREAD

**Discussion by Nelie-Taylor** [vikejs/vike#1672] [Discussion]
Hi, thank you for greate framework.
![image](https://github.com/vikejs/vike/assets/12018657/417bd95b-ef00-47da-867d-a6933ee48c45)
I have problem with initialization connect.
I have a config file at: ```server/configs/moleculer.ts```, i call it in ```pages/+data.ts```. But first time load page connect error (not initialization connect), next time, i can access to server (initialization in first time connect).
How to load file ```server/configs/moleculer.ts``` when start server?
I just try add to ```server/index.ts``` but it not work.
Thank you for support.

**Comment by brillout** [comment]
(Your issue doesn't seem related to Vike, but feel free to let me know if you think it is.)

**Reply by Nelie-Taylor** [reply]
thank you for reply.
I'm not sure this is Vike's problem.
I just use ```npm create vike@latest``` to create react.
Sumary my problem is first time access to page error when ```server/configs/moleculer.ts``` not initialized. I need initialized server/configs/moleculer.ts for start project.
